samples/kvsWebRTCClientMaster.c

```diff
@@ -10,6 +10,8 @@ INT32 main(INT32 argc, CHAR* argv[])
     UINT32 frameSize;
     PSampleConfiguration pSampleConfiguration = NULL;
     SignalingClientMetrics signalingClientMetrics;
+    BOOL useIot = FALSE;
```

Comment:
Let's move this to the same place as the other static values added to this sample in Samples.h